replacy/__init__.py

```diff
@@ -127,12 +122,19 @@ def get_predicates(self, match_hooks) -> List[Callable]:
             # predicate - filled template ex. succeeded_by_word("to")
             # will match "in addition to..." but not "in addition, ..."
             args = hook.get("args", None)
-            if args is not None:
-                # the match_hook needs arguments
-                pred = template(hook["args"])
+            kwargs = hook.get("kwargs", None)
+            if args is None:
+                if kwargs is None:
+                    # the match_hook is nullary
+                    pred = template()
+                else:
+                    pred = template(**kwargs)
+            elif type(args) == dict:
+                # should we force them to use kwargs?
```

Comment:
We could display a warning? 
Although - if this works - I would leave it. The less user need to know - the better (imo).

Reply:
Yeah part of me wants to get rid of `kwargs`, but part of me now hates `args`. So I guess I will leave for now

Reply:
Ok I changed this behavior and added a warning. Now if you want to call a function with a single dict as an argument, you can, by passing it to args (though it will warn you). If you want instead to call a function with many named args (a polyary? function? is that a word? multiary? I've only seen n-ary but that's dumb), you use `kwargs`